www/js/app/common/service/common-service.js

```diff
@@ -0,0 +1,44 @@
+var movieServices = angular.module('movieServices');
+movieServices.service('query', ['$resource', 'APP_CONSTANTS',  function($resource, APP_CONSTANTS) {
+	return {
+			'movieUpcoming' : $resource(APP_CONSTANTS.WS_URL +
```

Comment:
Since I'm not using a key that's not a valid Javasript identifier, the single quotes do not make a difference.